atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaHotspottingTest.java

```diff
@@ -125,18 +124,6 @@ public void testHardFailOnValidateOfIndexHotspottingSchema() {
         assertThatThrownBy(() -> getIndexHotspottingSchema().validate()).isInstanceOf(IllegalStateException.class);
     }
 
-    @Test
-    public void testFailToGenerateTableHotspottingSchema() throws IOException {
-        assertThatThrownBy(() -> getHotspottingSchema().renderTables(new TemporaryFolder().getRoot()))
-                .isInstanceOf(IllegalStateException.class);
-    }
-
-    @Test
-    public void testFailToGenerateIndexHotspottingSchema() throws IOException {
-        assertThatThrownBy(() -> getIndexHotspottingSchema().renderTables(new TemporaryFolder().getRoot()))
-                .isInstanceOf(IllegalStateException.class);
-    }
-
```

Comment:
May need a sanity check here. These two tests does not make any sense. Basically they succeed because `new TemporaryFolder().getRoot()` raises `IllegalStateException` which has nothing to do with what we want to test here. (I am not even sure what we want to test here. Function names are inadequate)

Reply:
Yeah, mostly agree with you? It seems like from past PRs people didn't expect to be able to render those schemas, but not clear to me why? (original PR where this got introduced: https://github.com/palantir/atlasdb/pull/1091). Tried to checkout the commit back from 2016 and see if it actually failed back then, but failed to run due to very old JDK and Gradle versions needed. 

These definitely don't seem to fail today at the rendering stage if we get rid of the problem with creating the temporary folder. I think we're OK with deleting the tests, since they don't seem to match real behaviour.